pkg/storage/stores.go

```diff
@@ -74,3 +74,11 @@ type SubjectAccessCapableStore interface {
 	GetRole(ctx context.Context, ref *corev1.Reference) (*corev1.Role, error)
 	ListRoleBindings(ctx context.Context) (*corev1.RoleBindingList, error)
 }
+
+type AlertingStore interface {
```

Comment:
unused?

Reply:
Unused, but will be used when I write the first AlertLog implementation using the gateway volume mount `/var/lib/alerting/`